### ISSUE TITLE
The Register-SecretVault cmdlet violates PowerShell’s best-practice coding rules.

### ISSUE THREAD

**Issue by wi-fr** [PowerShell/SecretManagement#81]
When strict mode is on, the Register-SecretVault cmdlet -- even though it completes what it is supposed to do -- generates a terminating error (“The property 'Path' cannot be found on this object. Verify that the property exists.”). Firstly, I would expect the Microsoft.PowerShell.SecretStore cmdlets to respect PowerShell’s best-practice coding rules and, secondly, if an error occurs anyway, the error to be non-terminating.

```
Set-StrictMode -Version Latest
try{
  Register-SecretVault -Name ‘MSStore' -ModuleName 'Microsoft.PowerShell.SecretStore'
  Register-SecretVault -Name 'KPStore' -ModuleName 'SecretManagement.Keepass'
} catch { $_ }
```

**Comment by SydneyhSmith** [comment]
Thanks @wi-fr we have a fix ready so you dont see the error

**Comment by PaulHigin** [comment]
Fixed for 1.0 RC1 release.
